src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/get-started/compatibility-requirements-aws-lambda-monitoring.mdx

```diff
@@ -34,6 +34,7 @@ older than 3.9.0.
 
 To minimize performance impact, we've taken a different approach with Java and .NET Core. New Relic provides
 [OpenTracing](https://opentracing.io/) SDKs for these runtimes. This approach requires a bit more code to integrate.
+For newer runtimes, such as .NET 6, [Trace your .NET Lambda functions with New Relic and OpenTelemetry](/docs/serverless-function-monitoring/aws-lambda-monitoring/opentelemetry/lambda-opentelemetry-dotnet/) is available.
```

Comment:
Recommend "you can trace your .NET lambda..." and remove the "is available" to remove the passive voice.